### DISCUSSION TITLE
Archived photos remain visible in shared albums of ohter users

### DISCUSSION THREAD

**Discussion by eeezae** [immich-app/immich#13332] [Q&A]
### The bug

When User A archives photos in a shared album, these archived photos remain visible in User B's interface. This behavior contradicts the expected result, where archived photos should not be accessible to other users in a shared album.

### The OS that Immich Server is running on

Debian

### Version of Immich Server

v1.117.0

### Version of Immich Mobile App

v1.117.0

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Not relevant
```

### Your .env content

```Shell
Not relevant
```

### Reproduction steps

1.User A creates an album with photos and shares it with User B.
2.User A archives some photos in the album.
3.In User B's interface (the shared album), the archived photos are still visible (both webpage and app).

**Comment by mmomjian** [comment]
Hello, thanks for the report. Archive photos are hidden from the main timeline view, not from albums. So this is the intended behavior.
